src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -368,6 +361,7 @@ const MissionEditView = ({ mission, toDetailsView, toListView, volunteers }) =>
                     <Grid item className={classes.fullWidth}>
                       <AddressInput
```

Comment:
change this to AddressAutocomplete. It should be the better version

Reply:
Do I have to use taht?  It doesn't work.  For instnace, it doesn't populate the address wiht a default value like "AddressInput" does ...

<img width="674" alt="Screen Shot 2020-05-22 at 4 43 11 PM" src="https://user-images.githubusercontent.com/14842313/82711605-7d7c4d80-9c4b-11ea-89aa-9b184004661a.png">

Reply:
it should work with defaultLocation props. The adressInput component is unfortunately outdated, with incorrect address and no map. You can check the AdressAutocompelete with default props in action by test order a foodbox

Reply:
For reference this is how it's being use for checkout https://github.com/factn/resilience-app/blob/master/src/app/page/Request/foodbox/DeliveryStep.jsx#L113

idk if the final design has the map view or but I guess we can worry about that later

Reply:
The 'final design' is miles behind. Can we pretty pretty please have the map preview turned on? For both address fields?